starters/starters.yaml

```diff
@@ -439,6 +439,15 @@
   tags: gridsome, starter, sanity, blog, portfolio, tailwind, tailwindcss, dark theme, responsive
   platforms: sanity
 
+- title: Gridsome Algrid
+  description: Algrid is a simple starter for creating blogs using Algolia and implementing infinite scroll.
```

Comment:
Sorry, can I be picky and get you to add `Algolia for search`? 🙂

Reply:
@thetre97 
OK! Thank you for your feedback i'm poor of english.. sorry.

Is this alright with you?
`description: Algrid is a simple starter for creating blogs using Algolia for search and implementing infinite scroll.`

Reply:
That looks great! I personally would add a comma after `search`, just to separate the two parts...

Reply:
OK! I finally understand! I'll correct it.
`description: Algrid is a simple starter for creating blogs using Algolia for search, and implementing infinite scroll.`

Reply:
Perfect 🙂